### DISCUSSION TITLE
How to test applications that use `prisma`?

### DISCUSSION THREAD

**Discussion by keinsell** [prisma/prisma#14114] [Q&A] [answered]
Documentation on website covers only `jest` and there is not much information how to mock/stub prisma client for tests with other frameworks.

**Comment by austincrim** [comment, marked as answer]
Hey @keinsell 👋,

While our docs only show a `jest` example, the basic process should be the same in other testing frameworks.

1. create a mock of the `PrismaClient` instance
2. provide mocked return values for each Prisma Client method that is used in the function you are testing

If you have specific questions about your testing setup, let me know the details and I will answer the best I can!

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
